pontoon/contributors/views.py

```diff
@@ -363,17 +338,8 @@ def edit_user_profile_locale_selector(request, username):
 @login_required(redirect_field_name="", login_url="/403")
 @require_POST
 @transaction.atomic
-def toggle_theme(request, username):
-    user = get_object_or_404(User, username=username)
-    if user != request.user:
-        return JsonResponse(
-            {
-                "status": False,
-                "message": "Forbidden: You don't have permission to edit this user",
-            },
-            status=403,
-        )
-
+def toggle_theme(request):
+    user = request.user
```

Comment:
This is unnecessary, the only usage is on line 346 below, which should be updated.